scene/3d/physics_body.cpp

```diff
@@ -915,6 +915,13 @@ String RigidBody::get_configuration_warning() const {
 
 	String warning = CollisionObject::get_configuration_warning();
 
+	if (get_scale() != Vector3(1.0, 1.0, 1.0)) {
+		if (warning != String()) {
+			warning += "\n\n";
+		}
+		warning += TTR("Scaling RigidBody is not supported by the physics engine and will lead to undefined behavior.\nYou may scale their parent node or mesh, but the RigidBody and its collision shape or polygon should have a scale of Vector3(1.0, 1.0, 1.0).\nThis design limitation may be lifted in a future release.");
```

Comment:
I *think* the scaling limitation for collision polygons/shapes was actually lifted in 3.0, but I prefer to err on the side of caution and discourage scaling shapes. It's much better to scale the parent, or the mesh.